include/engine/api/route_api.hpp

```diff
@@ -228,42 +228,50 @@ class RouteAPI : public BaseAPI
 
         std::vector<util::json::Object> annotations;
 
-        if (parameters.annotations_type != RouteParameters::AnnotationsType::None)
+        if (parameters.annotations_type != RouteParameters::AnnotationsType::None ||
+            parameters.annotations == true)
         {
+            auto ReqAnnotations = parameters.annotations_type;
+            if ((parameters.annotations == true) & (parameters.annotations_type &
+                RouteParameters::AnnotationsType::None))
```

Comment:
`(parameters.annotations_type & RouteParameters::AnnotationsType::None` will always be false, needs to be `==` instead of `&`.